recipe/meta.yaml

```diff
@@ -26,10 +26,16 @@ requirements:
   run:
     - wget
     - pkg-config
+    - cmake
+    - make
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
 
 test:
   commands:
     - kim-api-collections-management list
+    - kim-api-collections-management install CWD SW_StillingerWeber_1985_Si__MO_405512056662_006
```

Comment:
```suggestion
    - kim-api-collections-management install CWD SW_StillingerWeber_1985_Si__MO_405512056662_006  # [linux]
```

Reply:
By adding `# [linux]` at the end of the line the test is only executed on linux, as the model is not supported on Windows as far as I understand.

Reply:
@jan-janssen It is working on osx_arm64 according to the CI checks and I hope we can get it working on osx_64 with a small tweak